code/stable/template/SRS/HTML/Template_SRS.html

```diff
@@ -128,7 +128,95 @@ <h2>Table of Symbols</h2>
         <div id="Sec:TAbbAcc">
           <div class="subsection">
             <h2>Abbreviations and Acronyms</h2>
-            <p class="paragraph">There are no abbreviations and acronyms.</p>
+            <div id="Table:TAbbAcc">
```

Comment:
Are these abbreviations really all used in the Template? If they are not used, they should not be showing up here.